client/src/i18n/en-US.json

```diff
@@ -448,19 +448,19 @@
     },
     "editors": {
       "heading": "Editors",
-      "description": "Editors can perform any action on a submission, including assigning review coordinators.  They do not have access to the publication setup pages and cannot change publication settings.",
+      "description": "Editors can perform any action on a submission, including assigning review coordinators. They do not have access to the publication setup pages and cannot change publication settings.",
       "none": "An editor is not assigned to this publication.",
       "unassign": {
         "error": "An error occurred while attempting to unassign an editor.",
         "success": "{display_name} successfully unassigned as an editor."
       },
       "invite": {
         "NOT_UNIQUE": "Please select {display_name} from the search results dropdown.",
-        "invalid_email": "Pleaes provide a valid email address for the user you want to invite as an editor."
+        "invalid_email": "Pleaes provide a valid email address for the user you want to invite as editor."
       },
       "assign": {
         "unauthorized": "You are not authorized to assign editors. Please contact your administrator.",
-        "no_value": "Please provide the username, email, or name of the person you want to assign as an editor.",
+        "no_value": "Please provide the username, email, or name of the person you want to assign as editor.",
```

Comment:
I understand. I edited because it was my understanding that this message would be viewed when someone was attempting to assign a single user, not multiple users at once.

Reply:
That's correct, although I don't recognize why the previous version made that unclear. It only refers to one person as the subject of the sentence.